it/go/go.mod

```diff
@@ -4,36 +4,28 @@ go 1.20
 
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.2.2
-	github.com/microsoft/kiota-abstractions-go v1.2.0
 	github.com/microsoft/kiota-authentication-azure-go v1.0.0
 	github.com/microsoft/kiota-http-go v1.0.0
-	github.com/microsoft/kiota-serialization-form-go v1.0.0
```

Comment:
the serialization dependencies have been dropped, this is most likely because you ran the go install without a generated client and they got trimmed. I suggest dropping this change, dependabot will update to the latest for us on the main branch.